sphinx/templates/quickstart/conf.py_t

```diff
@@ -114,7 +114,7 @@ pygments_style = 'sphinx'
 # keep_warnings = False
 
 # If true, `todo` and `todoList` produce output, else they produce nothing.
-todo_include_todos = {{ ext_todo }}
+todo_include_todos = 'todo' in """{{ extensions }}"""
```

Comment:
This will misdetect if I enable an extension including "todo" in its name.

In addition, expanded conf.py will be very ugly.
```
todo_include_todos = 'todo' in """sphinx.ext.autodoc,sphinx.ext.doctest,sphinx.ext.todo"""
```